mtp_cashbook/apps/cashbook/context_processors.py

```diff
@@ -1,14 +1,23 @@
 from django.conf import settings
+from django.utils.safestring import mark_safe
 from django.utils.translation import gettext
 from mtp_common.context_processors import govuk_localisation as inherited_localisation
 
 
 def govuk_localisation(request):
+    global_header_text = gettext('Manage prisoners’ money')
+    if request.disbursements_available and hasattr(request, 'proposition_app'):
+        # NB: ensure proposition_app variables are html-safe
+        global_header_text += '</a> ' \
+                              '<span class="mtp-header-app">' \
+                              '<a href="%(url)s">%(name)s</a>' \
+                              '</span>' % request.proposition_app
+        global_header_text = mark_safe(global_header_text)
```

Comment:
gross

Reply:
the only way i could think to add an extra element without resorting to JS